components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -555,6 +558,7 @@ be lost if SCREAM_HACK_XML is not enabled.
 
     <physics inherit="atm_proc_group">
       <atm_procs_list>mac_aero_mic,rrtmgp</atm_procs_list>
+      <atm_procs_list COMPSET=".*DP-EAMxx">iop_forcing,mac_aero_mic,rrtmgp</atm_procs_list>
```

Comment:
I thought IOP was forcing dynamics, hence I was expecting to see it at the end of the step (right before dyn at the next step).

Reply:
The forcing is not specific to dynamics.  The placement of iop_forcing here is most consistent to where it was placed before, which was in homme post process right before coupling to physics (so right after dynamics).